sdk/src/wallet/types/balance.rs

```diff
@@ -80,18 +74,31 @@ pub struct BaseCoinBalance {
     pub(crate) voting_power: u64,
 }
 
-impl std::ops::AddAssign for BaseCoinBalance {
-    fn add_assign(&mut self, rhs: Self) {
-        self.total += rhs.total;
-        self.available += rhs.available;
-        #[cfg(feature = "participation")]
-        {
-            self.voting_power += rhs.voting_power;
-        }
-    }
+/// Mana and BIC fields for [`Balance`]
```

Comment:
```suggestion
/// Mana fields for [`Balance`]
```

Reply:
But then it's the same comment as the other one so do something about it :trollface:

Reply:
oop

Reply:
I think I'll wait until #1421 is merged just so I can use that new struct

Reply:
wait nvm it's a different pr and it's already merged lol